### DISCUSSION TITLE
How do I setup a Scan Response?

### DISCUSSION THREAD

**Discussion by taurr** [bluez/bluer#122] [Q&A]
Can we setup a specific Scan Response when using Bluer?

We need to setup the response for when a device does a Advertisement Scan on us (we need to return certain data). This can be done from the `bluetooth_peripheral` python libraries, but I can't find the corresponding functionality for `Bluer`.

**Comment by surban** [comment]
Could you provide more information about which BlueZ API is used by the Python library to provide that functionality? 

You can supply data with an advertisement using the [`manufacturer_data`](https://docs.rs/bluer/latest/bluer/adv/struct.Advertisement.html#structfield.manufacturer_data) and [`service_data`](https://docs.rs/bluer/latest/bluer/adv/struct.Advertisement.html#structfield.service_data) fields.

**Comment by taurr** [comment]
Obviously, now that I need to refind the example I found, I can't ☹️

I'm closing this issue, as I think I've basically found the answer myself, but in doing so, my question has morphed into several others...
